tests/tools/onert_train/src/args.cc

```diff
@@ -224,18 +253,21 @@ void Args::Initialize(void)
     ("epoch", po::value<int>()->default_value(5)->notifier([&](const auto &v) { _epoch = v; }), "Epoch number (default: 5)")
     ("batch_size", po::value<int>()->default_value(32)->notifier([&](const auto &v) { _batch_size = v; }), "Batch size (default: 32)")
     ("learning_rate", po::value<float>()->default_value(0.001)->notifier([&](const auto &v) { _learning_rate = v; }), "Learning rate (default: 0.001)")
-    ("loss", po::value<int>()->default_value(1)->notifier([&] (const auto &v) { _loss_type = v; }),
-        "Loss type\n"
-        "1: MEAN_SQUARED_ERROR (default)\n"
-        "2: CATEGORICAL_CROSSENTROPY")
-    ("loss_reduction_type", po::value<int>()->default_value(1)->notifier([&] (const auto &v) { _loss_reduction_type = v; }),
-        "Loss Reduction type\n"
-        "1: SUM_OVER_BATCH_SIZE(default)\n"
-        "2: SUM")
-    ("optimizer", po::value<int>()->default_value(1)->notifier([&] (const auto &v) { _optimizer_type = v; }),
-      "Optimizer type\n"
-      "1: SGD (default)\n"
-      "2: Adam")
+    ("loss", po::value<int>()
+      ->default_value(default_loss)
+      ->notifier([&](const auto& v){_loss_type = checkValidation("loss", valid_loss, v);}),
+      genHelpMsg("loss", valid_loss, default_loss).c_str()
+    )
+    ("loss_reduction_type", po::value<int>()
+      ->default_value(default_loss_rdt)
+      ->notifier([&](const auto &v){_loss_reduction_type = checkValidation("loss_reduction_type", valid_loss_rdt, v);}),
+      genHelpMsg("loss_reduction_tye", valid_loss_rdt, default_loss_rdt).c_str()
+    )
+    ("optimizer", po::value<int>()
+      ->default_value(default_optim)
+      ->notifier([&](const auto& v){_optimizer_type = checkValidation("optimizer", valid_optim, v);}),
+      genHelpMsg("optimizer", valid_optim, default_optim).c_str()
+    )
     ("metric", po::value<int>()->default_value(-1)->notifier([&] (const auto &v) { _metric_type = v; }),
```

Comment:
(note) 

After this, onert_train help messages slightly changed. 

```diff
➜ diff master_br_help current_br_help
<   --loss arg (=1)                       Loss type
<                                         1: MEAN_SQUARED_ERROR (default)
<                                         2: CATEGORICAL_CROSSENTROPY
<   --loss_reduction_type arg (=1)        Loss Reduction type
<                                         1: SUM_OVER_BATCH_SIZE(default)
<                                         2: SUM
<   --optimizer arg (=1)                  Optimizer type
<                                         1: SGD (default)
<                                         2: Adam
---
>   --loss arg (=1)                       Loss type (default: mean squared error)
>                                         1: mean squared error
>                                         2: categorical crossentropy
>   --loss_reduction_type arg (=1)        Loss Reduction type (default: sum over batch size)
>                                         1: sum over batch size
>                                         2: sum
>   --optimizer arg (=1)                  Optimizer (default: sgd)
>                                         1: sgd
>                                         2: adam
```


<details> 

* master branch help message 
	```
	onert_train
	
	Usage: /home/sseung/Workspace/ONE/master/Product/build/bin/onert_train [model path] [<options>]
	
	
	General options:
	  -h [ --help ]                         Print available options
	  --version                             Print version and exit immediately
	  --nnpackage arg                       NN Package file(directory) name
	  --modelfile arg                       NN Model filename
	  --path arg                            NN Package or NN Modelfile path
	  --export_path arg                     Path to export circle
	  --load_input:raw arg                  NN Model Raw Input data file
	                                        The datafile must have data for each input number.
	                                        If there are 3 inputs, the data of input0 must exist as 
	                                        much as data_length, and the data for input1 and input2 
	                                        must be held sequentially as data_length.
	  --load_expected:raw arg               NN Model Raw Expected data file
	                                        (Same data policy with load_input:raw)
	  -m [ --mem_poll ] arg (=0)            Check memory polling (default: false)
	  --epoch arg (=5)                      Epoch number (default: 5)
	  --batch_size arg (=32)                Batch size (default: 32)
	  --learning_rate arg (=0.00100000005)  Learning rate (default: 0.001)
	  --loss arg (=1)                       Loss type
	                                        1: MEAN_SQUARED_ERROR (default)
	                                        2: CATEGORICAL_CROSSENTROPY
	  --loss_reduction_type arg (=1)        Loss Reduction type
	                                        1: SUM_OVER_BATCH_SIZE(default)
	                                        2: SUM
	  --optimizer arg (=1)                  Optimizer type
	                                        1: SGD (default)
	                                        2: Adam
	  --metric arg (=-1)                    Metric type
	                                        Simply calculates the metric value using the variables 
	                                        (default: none)
	                                        0: CATEGORICAL_ACCURACY
	  --validation_split arg (=0)           Float between 0 and 1(0 < float < 1). Fraction of the 
	                                        training data to be used as validation data.
	  -v [ --verbose_level ] arg (=0)       Verbose level
	                                        0: prints the only result. Messages btw run don't print
	                                        1: prints result and message btw run
	                                        2: prints all of messages to print
	  --output_sizes arg                    The output buffer size in JSON 1D array
	                                        If not given, the model's output sizes are used
	                                        e.g. '[0, 40, 2, 80]' to set 0th tensor to 40 and 2nd 
	                                        tensor to 80.
	
	```

* current branch help message
	```
	onert_train
	
	Usage: /home/sseung/Workspace/ONE/current/Product/build/bin/onert_train [model path] [<options>]
	
	
	General options:
	  -h [ --help ]                         Print available options
	  --version                             Print version and exit immediately
	  --nnpackage arg                       NN Package file(directory) name
	  --modelfile arg                       NN Model filename
	  --path arg                            NN Package or NN Modelfile path
	  --export_path arg                     Path to export circle
	  --load_input:raw arg                  NN Model Raw Input data file
	                                        The datafile must have data for each input number.
	                                        If there are 3 inputs, the data of input0 must exist as 
	                                        much as data_length, and the data for input1 and input2 
	                                        must be held sequentially as data_length.
	  --load_expected:raw arg               NN Model Raw Expected data file
	                                        (Same data policy with load_input:raw)
	  -m [ --mem_poll ] arg (=0)            Check memory polling (default: false)
	  --epoch arg (=5)                      Epoch number (default: 5)
	  --batch_size arg (=32)                Batch size (default: 32)
	  --learning_rate arg (=0.00100000005)  Learning rate (default: 0.001)
	  --loss arg (=1)                       Loss type (default: mean squared error)
	                                        1: mean squared error
	                                        2: categorical crossentropy
	  --loss_reduction_type arg (=1)        Loss Reduction type (default: sum over batch size)
	                                        1: sum over batch size
	                                        2: sum
	  --optimizer arg (=1)                  Optimizer (default: sgd)
	                                        1: sgd
	                                        2: adam
	  --metric arg (=-1)                    Metric type
	                                        Simply calculates the metric value using the variables 
	                                        (default: none)
	                                        0: CATEGORICAL_ACCURACY
	  --validation_split arg (=0)           Float between 0 and 1(0 < float < 1). Fraction of the 
	                                        training data to be used as validation data.
	  -v [ --verbose_level ] arg (=0)       Verbose level
	                                        0: prints the only result. Messages btw run don't print
	                                        1: prints result and message btw run
	                                        2: prints all of messages to print
	  --output_sizes arg                    The output buffer size in JSON 1D array
	                                        If not given, the model's output sizes are used
	                                        e.g. '[0, 40, 2, 80]' to set 0th tensor to 40 and 2nd 
	                                        tensor to 80.
	```
</details>

Reply:
If invalid input is given, it shows below messages. 

```bash 
➜ ~/Workspace/ONE_second/Product/build/bin/onert_train \
--modelfile mnist_with_meta.circle \
--load_input:raw pool_mnist_model/mnist_x_train.1000.bin \
--load_expected:raw pool_mnist_model/mnist_y_train.1000.bin \
--loss_reduction_type 3   
Model Expected Filename pool_mnist_model/mnist_y_train.1000.bin
Model Input Filename pool_mnist_model/mnist_x_train.1000.bin
loss_reduction_type 3 is unsupported argument
```